nanodbc/nanodbc.cpp

```diff
@@ -2028,6 +2041,27 @@ class statement::statement_impl
             NANODBC_THROW_DATABASE_ERROR(stmt_, SQL_HANDLE_STMT);
     }
 
+    void set_param_descr(
```

Comment:
I am not convinced this name is a good name.

I would suggest to consider something that is not based on abbreviations, that is more self-descriptive, raises less questions, hence clearer e.g. `describe_parameters`. What do you think?

Reply:
I don't mind abbreviations in general, but yeah I agree that `describe_parameters` reads a bit better.